raster-foundry/raster-foundry
# Update Session complete endpoint

## Overview

This PR updates the task session complete endpoint so that it disables the labels from the previous sessions of the same task and promotes the current task session.

### Checklist

- ~Swagger specification updated~
- ~New tables and queries have appropriate indices added~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
- ~Any new endpoints have scope validation and are included in the integration test csv~

## Testing Instructions

- CI should pass as new property tests were added for the dao methods
- Follow steps here: https://github.com/raster-foundry/raster-foundry/pull/5522 to create a task session of task `A`, post some labels to the task and `complete` the task session. Make sure the labels are posted to the task and the session and the labels are set to `is_active = true`
- Create a new session of the same task `A`. Post some labels to the session. `complete` the session. Make sure that labels from the session in step two are set to `is_active = false`, and that the labels from the session in this step are posted correctly with the `is_active = true` field

Closes https://github.com/raster-foundry/groundwork/issues/1433
